### DISCUSSION TITLE
Auto Queue always start from Ksampler

### DISCUSSION THREAD

**Discussion by grzechuum** [Comfy-Org/ComfyUI_frontend#1259] [Q&A]
Hi everyone
I've made change to Clip Text Encode (prompt) to get prompt from mysql DB (prompt is always random), but have problem with Auto Queue feature thats always loop from Ksampler and run this node with the same prompt. How to change AutoQueue to start from Clip Text Encode (prompt)?

Node must be change to get to queue, how to force change ?

**Comment by LukeG89** [comment]
I'm not really sure if this could be helpful, but I know there is a node in [ComfyUI-Impact-Pack](https://github.com/ltdrdata/ComfyUI-Impact-Pack) called `Execution Order Controller`.
I've never used it, but by its name seems to work for that purpose (?)
